.github/workflows/ci-dgraph-e2e-net.yml

```diff
@@ -0,0 +1,42 @@
+name: ci-dgraph-e2e-net-tests
+
+on:

```

Comment:
we should copy these from the CI workflows in the dgraph repo. could you check please?

Reply:
Copy what exactly? The name? `on:` ?

Reply:
sorry, I meant the whole workflow except the part that needs to be different. https://github.com/dgraph-io/dgraph/tree/main/.github/workflows. dgo works too I guess.

Reply:
I changed both to match the dgo workflows as closely as possible. The steps need to be different, but the rest is the same.